css/style.css

```
@@ -0,0 +1,596 @@
+  color: #ec5242;
+  border: 3px solid #ec5242;
+  font-weight: 600;
+}
+
+.navbar .container-fluid .btn-outline-success:hover {
+  color: white;
+  background-color: #ec5242;
+}
+
+.first-section {
+  background-image: url(../images/first-section.png);
+  background-repeat: no-repeat;
+  margin-top: -5.2rem;
+  position: absolute;
+  background-size: cover;
+  width: 100%;
+  height: 669px;
+  background-position: bottom;
+}
+
+.first-section .first-section-container {
+  margin: 20px;
+}
+
+.first-section .first-section-container .second-heading {
+  color: #ec5242;
+  padding-top: 4rem;
+  font-family: "lato", sans-serif;
+}
+
+.first-section .first-section-container .first-section-paragraph .first-paragraph {
+  font-size: 12px;
+  font-weight: 600;
+  margin-top: 58px;
+  border: 5px solid white;
+  color: #272a31;
+  background: #f7f7f7;
+}
+
+.first-section .first-section-container .event-details {
+  font-size: 13px;
+  font-weight: 500;
+  font-family: "Cocogoose", sans-serif;
+}
+
+.first-section .first-section-container h1 {
+  font-size: 37px;
+  font-family: "Cocogoose", sans-serif;
+  color: #ec5242;
+}
+
+.navbar-toggler-icon {
+  height: 1em;
+  width: 1em;
+  border: none;
+  outline: none;
+}
+
+.first-section .first-section-container .event-details h3 {
+  font-size: 1.3rem;
+  font-weight: 400;
+}
+
+.second-section {
+  margin-top: 42rem;
+  background-image: url(../images/repeat-bg.png);
+  color: white;
+}
+
+.second-section .second-section-container .second-section-heading {
+  color: white;
+  display: flex;
+  justify-content: center;
+  padding: 30px;
+}
+
+.second-section .second-section-container .program-list {
+  display: grid;
+  grid-gap: 10px;
+  padding: 10px;
+}
+
+.section-speakers-heading h2 span {
+  border-bottom: 3px solid #ec5242;
+  padding-bottom: 10px;
+}
+
+.events h4 span {
+  border-bottom: 2px solid #ec5242;
+  padding-bottom: 5px;
+}
+
+.partners-section .partners-heading span {
+  border-bottom: 2px solid #ec5242;
+  padding-bottom: 5px;
+}
+
+.second-section .second-section-container .second-section-heading span {
+  border-bottom: 3px solid #ec5242;
+  padding-left: 5px;
+  padding-bottom: 10px;
+}
+
+.second-section .second-section-container .program-list .icons {
+  width: 37px;
+  height: 40px;
+  margin-top: 7px;
+}
+
+.second-section .second-section-container .program-list .flex-box {
+  display: inline-flex;
+  flex-wrap: nowrap;
+  justify-content: space-around;
+  background-color: #272a31;
+  padding: 20px;
+}
+
+.section-speakers-heading h2 {
+  font-family: "lato", sans-serif;
+  font-weight: 600;
+}
+
+.second-section .second-section-container .program-list .first-block h2,
+.second-section .second-section-container .program-list .second-block h2,
+.second-section .second-section-container .program-list .third-block h2,
+.second-section .second-section-container .program-list .fourth-block h2,
+.second-section .second-section-container .program-list .fifth-block h2 {
```

All these blocks share the same styling. If they had been given the same class in the index.html for example program-item the style calling could be quite simple. This helps for DRY CSS.
```suggestion
.program-block h2 {
```